example/storybook/src/components/Overlay/AlertDialog/index.stories.mdx

```diff
@@ -409,6 +420,82 @@ The following are the lists of all the libraries and packages the component reli
 - `@gluestack-ui/transitions`
 - `@react-native-aria/dialog`
 
+### Examples
+
+#### AlertDialog
+
+<CodePreview
+  showComponentRenderer={true}
+  showArgsController={false}
+  metaData={{
+    code: `
+        function App(){
+            const [showAlertDialog, setShowAlertDialog] = React.useState(true);
+            return (
+                <Wrapper>
+                  <Center h={300} >
+                      <Button onPress={() => setShowAlertDialog(true)}>
+                         <Button.Text>Click me</Button.Text>
+                      </Button>
+                  </Center>
+                  <AlertDialog
+                          isOpen={showAlertDialog}
+                          onClose={() => {
+                              setShowAlertDialog(false);
+                          }}
+                          bgColor="white">
+                    <AlertDialog.Content bgColor="white">
+                      <AlertDialog.Body p='$5'>
+                        <HStack space="md" alignItems='center'>
+                          <AlertTriangle color="#B91C1C" size={16}/>
```

Comment:
Add a comment that you are importing external icons from lucid-icons.

Reply:
done